http/vertx/src/test/java/io/quarkus/ts/vertx/AbstractVertxIT.java

```diff
@@ -21,13 +21,19 @@
 import io.restassured.specification.RequestSpecification;
 import io.vertx.core.Vertx;
 import io.vertx.core.http.HttpClient;
+import io.vertx.core.http.HttpClientOptions;
+import io.vertx.core.http.HttpClientResponse;
 import io.vertx.core.http.HttpMethod;
+import io.vertx.core.http.HttpVersion;
+import io.vertx.core.json.JsonObject;
 
 public abstract class AbstractVertxIT {
 
     @QuarkusApplication
     static final RestService service = new RestService();
 
+    private String localhost = service.getHost().replace("http://", "");
```

Comment:
Wouldn't `io.quarkus.test.bootstrap.Protocol#NONE` do the trick? Sorry, didn't try it, but this feels like a hack.

Reply:
Well, I know what you feel about the "hack" but I've checked Protocol class and NONE option but that doesn't do the trick. If you know any quick way in quarkus test framework please tell me and I will change it. Thanks

Reply:
Nah, not that important. I'll try it when I find a little time, but your explanation is ok.

Reply:
@jcarranzan I take it back, `service.getURI(Protocol.NONE).getHost()` does the trick.

Reply:
please use it instead of what I call the hack :-)

Reply:
oh I see, it has been changed as you mentioned, thank you @michalvavrik !